doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+
+•	Network connectivity for L3 tenants, GWs, SDS, etc
+
+•	Spine-Leaf Architecture ( 3 Tier or 5 Tier CLOS architecture )
+
+•	Automation of the programmable newtorking fabric
+
+•	Variety of Underlay protocols that can be used as a transport layer within the physical fabric
+
+•	Variety of Overlay protocols
+
+•	SDN integration with Infrastructure Management
+
+•	SRIOV workload interworking
+
+•	DPDK workload interworking
+
+•	Service Function Chaining (SFC ) and how it can be achieved
+
+•	New networking realisation technologies ( SmartNIC , FPGA ,..)
+
+•	Inter K8s POD connectivity and CNI
+
+•	Networking aspects of migration from IaaS to CaaS (from the VNF paradigm to the CNF paradigm)
+
+•	Coexistence of IaaS and CaaS and protracted parallel operations
+
+•	Version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration with a wide selection of fabric implementations
 
-> _Placeholder for the Executive Summary (targeting Baldy for first draft). **If you are interested in writing this content, please contact the RM lead.**_
 
 <a name="5.1.2"></a>
-### 5.1.2 Networking Strategy Objectives
-
-This section catalogs CNTT's high-level objectives for the Network Fabric Strategy. 
-
-> List needs to be prioritized; expect additional objectives to be added, as they arise. This represents the _What_, not the _How_.
-
-1. The implementation of Networking inside the HW Layer should not be visible to the VNF/CNF and should preferably not even be visible to the IaaS/CaaS
-1. Provide networks for L3 tenant, GWs, SDS, etc.
-1. Cleanly decouple interface/reference points between CNTT constituencies
-1. Provide interoperability at layer demarcation/reference points within the cloud infrastructure. Ex.:
-   * any RA couples to RM
-   * like RI couples to RA
-   * like VIs (Vendor's Implementations) couple to RA
-   * Operators can design or procure a compatible fabric
-1. Concurrently supports containerized and virtualized coexistence for VNF->CNF cutovers, as well as protracted parallel operations
-1. Provide a version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration w/ a wide selection of fabric implementations
-1. Provide ability for any number of Operator-specific fabrics to power CNTT cloud infrastructure
-1. Enable RC's ability to realize mandated OVP qualification deliverables
-1. Unambiguously document the responsibilities of each CNTT constituency
-1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
-1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
-1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
-
-> _**Editor's Note:** Consider moving objectives to a table_
+### 5.1.2 Networking Principles & Strategy
```

Comment:
Can we move this to a separate PR please? 
The title for this is to define the scope. 
Also this text is already forking from other PRs.